### PR TITLE
Make popover support shadow DOM

### PR DESCRIPTION
**Pull request by kilobyte2007**
When using Keen-UI components in the shadow DOM, we need the popovers to be appended only to the root node of the Shadow DOM.

### CODE REVIEW

File: src/UiPopover.vue [hunk 1]
```diff
@@ -113,12 +113,14 @@ export default {
                 return;
             }
 
+            const body = this.triggerEl.getRootNode() === document ? document.body : this.triggerEl.getRootNode();
```
Comment (JosephusPaye) [inline src/UiPopover.vue:116]: A comment explaining why we're doing this would be nice.

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
Looks good. Just one request.

File: src/UiPopover.vue (hunk 1 above)
Comment (kilobyte2007) [reply src/UiPopover.vue:117]: Indeed. Added a comment.

**Review by JosephusPaye — APPROVED** [review]

**Comment by kilobyte2007** [comment]
Hey @JosephusPaye, can you please merge this?

**Comment by JosephusPaye** [comment]
Thanks! Released in [v1.3.2](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.3.2)

**Pull request merged**